controller/handler-test/compliance-values.js

```diff
@@ -0,0 +1,105 @@
+'use strict';
+
+const test = require('ava');
+const sinon = require('sinon');
+const proxyquire = require('proxyquire');
+const QueryTypes = {
+    SELECT: 'select'
+};
+
+test.cb('when patients exist in trial', (t) => {
+    const model = sinon.stub();
```

Comment:
the `model` stub is not needed, the `query` stub is needed and needs to return something.
the missing `query` return is causing the tests to fail currently.